python/tank/descriptor/bundle_cache_usage/__init__.py

```diff
@@ -0,0 +1,121 @@
+from ... import LogManager
+from ...util import LocalFileStorageManager
+
+# Debugging options
+USE_PYDEVD = False
```

Comment:
this is all temp stuff, right?

Reply:
Yes, my last commit will be about deleting that stuff which I really need until then. The usual SG logger is simply to noisy when trying to isolate specific operations. Also, this is simpler for me working with PyCharm or Liclipse.